src/sentry/api/serializers/models/organization.py

```diff
@@ -174,17 +173,43 @@ def get_attrs(
             a.organization_id: a
             for a in OrganizationAvatar.objects.filter(organization__in=item_list)
         }
-        auth_providers = {
-            a.organization_id: a for a in AuthProvider.objects.filter(organization__in=item_list)
+
+        configs_by_org_id: Mapping[int, ApiOrganizationAuthConfig] = {
+            config.organization_id: config
+            for config in auth_service.get_org_auth_config(
```

Comment:
This batches several related 'auth config' objects related to an organization, making this serialization process possible.